### DISCUSSION TITLE
When will 4.4 be released?

### DISCUSSION THREAD

**Discussion by StFS** [jekyll/jekyll#9576] [General]
I just spent quite some time updating my Jekyll site to the latest version because I needed to be able to highlight multiple lines in a code block (https://jekyllrb.com/docs/liquid/tags/#marking-specific-lines).

Little did I know that this feature is marked with being available in version 4.4 but the latest version now available is 4.3.

The documentation for this feature is well over a year old (it was changed to be marked as available in 4.4 in https://github.com/jekyll/jekyll/pull/9184) but as I said, this is a feature that really is not available.

So I ask, when is 4.4 expected to become available? And is it a good idea to include documentation about features that aren't available in published versions on the main documentation site? Shouldn't it at least be clearly marked as an "unreleased" version (and perhaps collapsed by default)?

Sorry for the (bit of a) rant. I appreciate open source projects and Jekyll is awesome. Just frustrated over my own haste here, but at the same time do think that this could possibly have been presented more clearly and probably saved me some time if it had been.

**Comment by perseoGI** [comment]
When will Jekyll 4.4 will be released?
We also need to highlight specific lines, which is done in https://github.com/jekyll/jekyll/pull/9138

This was merged almost 2 years ago, I think is time to move it forward. 
Please, thank you in advance!

**Comment by dijonkitchen** [comment]
Should be resolved now: https://github.com/jekyll/jekyll/releases

**Comment by alebenitez** [comment]
No te preocupes no hay problema solo te pido un favor entre vos y yo y
elimínalo decime Cuánto por favor me van a devolver Porque estoy muy
necesitada de dinero te lo agradecería un montón y cómo toca ser para ir a
cobrarlo

**Comment by alebenitez** [comment]
Discúlpame no entendí mucho Pero bueno tú me estás hablando del banco
verdad

**Comment by alebenitez** [comment]
Yo termino conversar contigo y Ya borró todo pero el miércoles yo tengo
algo para cobrar Entonces ya dónde tengo que dirigirme no entiendo por qué
